### PR TITLE
Created a Badge component

### PR DESCRIPTION
**Pull request by pritam-kr**
- Badge with image
- Badge with icon
- Badge with text

 
Screenshot (121)

Screenshot (122)

### CODE REVIEW

File: components/Alert.css [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+     background-color: var(--primary-color);
+     color: var(--headings-color);
  }
- .successfull-alert{
-    background-color: #84F1C9;
-    color: #068154;
+ 
+ .successfull-alert {
+     background-color: #84F1C9;
+     color: #068154;
```
Comment (mimansha-swarup) [inline components/Alert.css:16]: I will suggest making variable of colors like you did for primary-color

File: components/badge.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    right: 1em;
+}
+
+.status-away {
+    background-color: yellow;
+    right: .3em;
+}
+
+.status-distrube {
+    background-color: rgb(117, 117, 117);
```
Comment (KoushikKintali) [inline components/badge.css]: For colors it is advised to declare a variable & use it here

**Review by KoushikKintali — APPROVED** [review]
Looks good, just review the comment

**Review by mimansha-swarup — COMMENTED** [review]
except this all looks good

**Review by horsemaker — COMMENTED** [review]
Other than the part already mentioned about using variables for colors. This looks fine to me! 👍

File: components/Alert.css (hunk 1 above)
Comment (pritam-kr) [reply components/Alert.css:16]: I saved all colors in variables.  I'll take care of it next time. Thank you @mimansha-swarup @KoushikKintali @horsemaker

**Review by manojasarsa — APPROVED** [review]
Looks clean! Good to go👍

**Pull request merged**